### PR TITLE
DOC-12079 Lite links and App Services intro

### PR DESCRIPTION
**Pull request by RichardSmedley**
Interim landing page, giving necessary App Services and Lite info to mobile devs on the Capella site.

Note, once the Server side has similar dev content, this will be replicated there as a further interim developer page -
until all of the Information Architecture changes are completed.

image

### CODE REVIEW

File: modules/mobile-guides/pages/intro.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 = Build Mobile Apps
 :page-role: tiles -toc
-:description: Use App Services in combination with Couchbase Mobile to develop mobile apps.
+:description: Use App Services in combination with Couchbase Lite to develop mobile apps.
```
Comment (idulo) [inline modules/mobile-guides/pages/intro.adoc]: Suggestion: Use App Services in Capella in combination with Couchbase Lite embedded NoSQL database to develop mobile apps.

File: modules/mobile-guides/pages/intro.adoc [hunk 2]
```diff
@@ -20,19 +20,26 @@ ifdef::basebackend-html[]
 ++++
 endif::[]
 
+[abstract]
 {description}
 
 include::develop:partial$escape-hatch.adoc[]
+Couchbase Mobile consists of Couchbase Lite -- an embedded, offline-first database for edge devices -- synced peer-to-peer, or synchronized to your Couchbase Cluster via App Services.
```
Comment (idulo) [inline modules/mobile-guides/pages/intro.adoc]: Couchbase Mobile also consists of Sync Gateway for on prem users.

Comment (idulo) [reply modules/mobile-guides/pages/intro.adoc]: You can either use Capella App Services (managed) or Sync Gateway (self-managed/on prem) with Couchbase Lite for cloud-to-edge sync. Developers can also build offline apps just with peer-to-peer as indicated, so just with Lite.

Comment (RichardSmedley) [reply modules/mobile-guides/pages/intro.adoc]: Acknowledge your point on SGW, but atm the non-Capella version of this page is still in development by @simon-dew 
When that is done, each version can mention and link to the other.
For now I'll change this to add something parenthetic about SGW.  :+1:

Comment (RichardSmedley) [reply modules/mobile-guides/pages/intro.adoc]: That addendum in parenthesis can become a link (on Sync Gateway) once the non-Capella developer doc mirror of this page is created.

**Comment by RichardSmedley** [comment]
Merging as there have been no further comments.
We can certainly return to this page and make further improvements before the discussed IA changes kick in.

**Pull request merged**